gateway/1.4/modules/ROOT/pages/_partials/configuration-task-list.adoc

```diff
@@ -0,0 +1,27 @@
+//tag::configuration-task-list-replicas[]
+* xref:flex-{page-mode}-rep-run.adoc[Configure Replicas] - Anypoint Flex Gateway enables you to create replicas, which are instances of Flex Gateway. By default, Flex Gateway runs as a single replica. Running multiple replicas enables scaling, load balancing, and high availability.
+//end::configuration-task-list-replicas[]
+
+//tag::configuration-task-list-tls[]
+* xref:flex-{page-mode}-tls-config.adoc[Configure TLS Context] - Flex Gateway enables you to configure a TLS context to support regular TLS and mTLS in both inbound and outbound directions.
+//end::configuration-task-list-tls[]
+
+//tag::configuration-task-list-shared-storage[]
+* xref:flex-{page-mode}-shared-storage-config.adoc[Configure Shared Storage] - Flex Gateway enables you to configure shared storage for distributed caching and rate limiting policies.
+//end::configuration-task-list-shared-storage[]
+
+//tag::configuration-task-list-forward-proxy[]
+* xref:flex-{page-mode}-forward-proxy.adoc[Configure a Forward Proxy] - Flex Gateway enables you to configure a forward proxy to route external HTTP connections through a proxy connection.
+//end::configuration-task-list-forward-proxy[]
+
+//tag::configuration-task-list-apim-api[]
+* xref:flex-{page-mode}-manage-public-api.adoc[Configure Using the API Manager API] - Flex Gateway enables you to configure and manage Flex Gateway via the API Manager API, a command line tool used to communicate with *API Manager*.
```

Comment:
Is the API a command line tool?

Reply:
@hannanelson Actually the tool name is "API Manager API", but I'm not sure how to express it here. Does it need to be in quotes? In bold?

Reply:
Either refer to it as "the API Manager API tool" or "API Manager API" (after you've introduced it).

Also, who decided on that for a tool name? :>

Reply:
"Configure Using" is a little strange. Configure with? Use the xxx to Configure?